docs/ansible/roles/cron/defaults-detailed.rst

```diff
@@ -361,3 +361,15 @@ Execute a custom script every week, as the ``backup`` user:
            content: |
              #!/bin/bash
              touch /tmp/weekly-result
+
+Disable email notification for one job only:
+
+.. code-block:: yaml
+
+   cron__jobs:
+
+     'script-without-mail':
+       special_time: 'hourly'
+       job: '/usr/local/lib/hourly-job'
+       environment:
+         - MAILTO: ''
```

Comment:
Ah, another problem - the `environment` parameter needs to be a dictionary, not a list.

Reply:
You mean: remove dash?

```
environment:
  MAILTO: ''
```

?

Reply:
Yes, exactly. You could include additional variables to make it easier to notice.